app/src/main/java/com/github/fribourgsdp/radio/MainActivity.kt

```diff
@@ -2,18 +2,24 @@ package com.github.fribourgsdp.radio
 
 
 
+import android.content.ContentValues
 import android.content.Intent
 import android.os.Bundle
+import android.util.Log
 import android.widget.Button
 import androidx.appcompat.app.AppCompatActivity
+
+//import com.github.fribourgsdp.radio.databinding.ActivityMainBinding
+
 import android.widget.ImageButton
 
 class MainActivity : AppCompatActivity() {
+    private val db = FirestoreDatabase()
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_main)
         val fireBaseButton = findViewById<Button>(R.id.FireBaseButton)
-        fireBaseButton.setOnClickListener{startActivity(Intent(this,FireBaseTestActivity::class.java))}
+        fireBaseButton.setOnClickListener{startActivity(Intent(this,CreateUserProfileActivity::class.java))}
```

Comment:
Please change this button name :)

Reply:
This was solved when merging from master.